src/FffGcodeWriter.cpp

```diff
@@ -677,52 +617,116 @@ void FffGcodeWriter::processDraftShield(SliceDataStorage& storage, GCodePlanner&
         }
     }
 
-    gcode_layer.addPolygonsByOptimizer(storage.draft_protection_shield, &storage.skirt_brim_config[0]); //TODO: Skirt and brim configuration index should correspond to draft shield extruder number.
+    gcode_layer.addPolygonsByOptimizer(storage.draft_protection_shield, &gcode_layer.configs_storage.skirt_brim_config[0]); //TODO: Skirt and brim configuration index should correspond to draft shield extruder number.
 }
 
-std::vector<int> FffGcodeWriter::calculateExtruderOrder(SliceDataStorage& storage, int current_extruder)
+void FffGcodeWriter::calculateExtruderOrderPerLayer(const SliceDataStorage& storage)
```

Comment:
No it's parsed as 'calculate extruder_order_per_layer`.
Changing calculateExtruderOrder to calculateLayerExtruderOrder